dvc/command/ls/__init__.py

```diff
@@ -54,14 +54,12 @@ def add_parser(subparsers, parent_parser):
         formatter_class=argparse.RawTextHelpFormatter,
     )
     list_parser.add_argument(
-        "url",
-        help="Supported urls:\n"
-        "/path/to/file\n"
```

Comment:
Not sure why are we getting rid of these useful examples.

Reply:
Yeah for consistency with get and import and so the output here is shorter. We could go the opposite way and list all the examples in all these commands if that's preferable. Or we could add a link to the cmd ref in each one. Please lmk, happy to change it.